### DISCUSSION TITLE
__slots__ in machine.py

### DISCUSSION THREAD

**Discussion by mrechte** [missionpinball/missionpinball#119] [Q&A]
Hello.

I'm trying to familiarize myself with the source code, so please bear with me if my question sounds stupid.

Class MachineController is based on LogMixin. LogMixin defines a slot named "log". Why MachineController redefines this slot ?

This is not conform with what is stated at the end of this document: "child subclasses... which should only contain names of additional slots" 

https://wiki.python.org/moin/UsingSlots

**Comment by toomanybrians** [comment]
Good find! This is probably a mistake. Feel free to submit a PR to fix it. Thanks!
